world.js

```diff
@@ -958,10 +992,10 @@ world.addEventListener('trackedobjectadd', async e => {
     const minimapObject = minimap.addObject(mesh);
     mesh.minimapObject = minimapObject;
 
-    if (token.owner.address && token.owner.monetizationPointer && token.owner.monetizationPointer[0] === "$") {
+    if (contentId && instanceId && token.owner.address && token.owner.monetizationPointer && token.owner.monetizationPointer[0] === "$") {
```

Comment:
I don't think it ever makes sense for there to not be these ids; that would be a bug.

Reply:
502e348